doozerlib/rpmcfg.py

```diff
@@ -68,15 +69,13 @@ def set_nvr(self, version, release):
         self.version = version
         self.release = release
         self.tag = '{}-{}-{}'.format(self.config.name, self.version, self.release)
-        self.vtag = 'v{}-{}'.format(self.version, self.release)
 
     def create_tag(self, scratch):
         if not self.tag:
             raise ValueError('Must run set_nvr() before calling!')
 
         with Dir(self.source_path):
-            exectools.cmd_assert('git tag {}'.format(self.tag))
-            exectools.cmd_assert('git tag {}'.format(self.vtag))
```

Comment:
why are you getting rid of vtag?

Reply:
I introduced vtag to support `openshift` RPM which has a custom tagger to do that (`openshift-ansible` may as well). Basically because I didn't understand all of doozer's interactions with tito. Since this now uses `openshift`'s tito config correctly it's no longer needed.